plugin.js

```diff
@@ -13,19 +14,22 @@ export default (ctx) => {
 
   const { isDev, isClient, isServer, app, route, beforeNuxtRender, store } = ctx
 
-  <% Object.keys(options.networkInterfaces).forEach((key) => { %>
-    let networkInterface = require('<%= options.networkInterfaces[key] %>')
-    networkInterface = networkInterface.default(ctx) || networkInterface(ctx)
+  <% Object.keys(options.clientConfigs).forEach((key) => { %>
+    let client = require('<%= options.clientConfigs[key] %>')
+    // es6 module default export or not
+    client = client.default(ctx) || client(ctx)
 
     const opts = isServer ? {
-        ssrMode: true
+        ssrMode: true,
+        cache: new InMemoryCache()
     } : {
-      initialState: window.__NUXT__ ? window.__NUXT__.apollo.<%= key === 'default' ? 'defaultClient' : key %> : null,
+      initialState: window.__NUXT__ ? window.__NUXT__.apollo.<%= key === 'default' ? 'defaultLink' : key %> : null,
```

Comment:
There is no `intialState` anymore. See [the docs](https://github.com/Akryum/vue-apollo/tree/next#creating-the-apollo-clients).

Reply:
dope